robot/Cumulus/resources/NPSP.py

```diff
@@ -868,7 +868,7 @@ def wait_for_batch_to_complete(self, path, *args, **kwargs):
                     self.selenium.wait_until_element_is_visible(locator)
                     break
                 except Exception:
-                    time.sleep(15)    
+                    time.sleep(20)
```

Comment:
This results in waiting for more than six minutes. Is that really what you intend to do?